client/src/pages/RoomPage/index.tsx

```diff
@@ -298,7 +319,7 @@ function RoomPage({ name }: { name: string }): JSX.Element {
           {announcement}
         </Notification>
         <AvatarCanvas
-          selfUserInfo={selfUserInfoRef.current}
+          selfUserInfo={userInfosRef.current[socket.id]}
```

Comment:
If this was changed to `selfInfo`, would it cause stale closure?

Reply:
I tried it locally and seems to work with `selfInfo`

Reply:
It shouldn't since it's being consumed by a react component, nice catch thanks!